index.html

```diff
@@ -1211,9 +1226,11 @@ <h3>Algorithm</h3>
           <var>term</var> MUST NOT be a <a>keyword</a> and a
           <a data-link-for="JsonLdErrorCode">keyword redefinition</a>
           error has been detected and processing is aborted.</li>
-        <li class="changed">If the <var>active context</var> has an existing
-          <a>term definition</a> for <var>term</var> which is protected, processing is aborted;
-          processors SHOULD issue a warning that an attempt was made to redefine a protected term.</li>
+        <li class="changed">If the <var>from term</var> is <code>false</code>
+          and <var>active context</var> has an existing
+          <a>term definition</a> for <var>term</var> which is protected,
+          a <a data-link-for="JsonLdErrorCode">protected term redefinition</a> error has been detected,
+          and processing is aborted.</li>
```

Comment:
The one thing we might want to modify here is to allow for "redefinitions" that cause the *exact same term definition* to occur. There's really no reason disallow that from an authorship perspective -- and people might be surprised that they can't combine two contexts that share a common base context with protected terms. On the other hand, this would complicate implementation.

@davidlehn -- since you were in the code most recently, would this be a tough thing to support?

Reply:
Either way, I don't think that this should hold up the PR.

Reply:
We could say that an attempt to redefine a term definition that is exactly equivalent to the existing definition is allowed, needing to define what we mean by equivalent, but it seems an unnecessary complication to me.